benchmarks/AiDotNet.PyTorchParity/pytorch/compare.py

```
@@ -0,0 +1,98 @@
+Usage:
+    python compare.py ../results/aidotnet.json ../results/pytorch.json
+"""
+
+from __future__ import annotations
+
+import argparse
+import json
+from pathlib import Path
+
+
+def _get(d: dict, *names: str, default=None):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_get`

(ANN202)